### ISSUE TITLE
Can't launch notebook after `nest_asyncio` upgraded to 1.5.2

### ISSUE THREAD

**Issue by tailaiw** [jupyter/notebook#6242]
<!--
BEFORE YOU OPEN AN ISSUE, PLEASE READ THIS

Hello! Thank you for using Jupyter Notebook. We're glad you're here.

Right now, you're opening an issue. Before you do, let's make sure this is the right place to post your question/issue.

First, it's important to know that Jupyter Notebook development has moved into a phase of maintenance-only. There are very few people with limited time maintaining this repository. This means, we won't likely accept new features here. Instead, we recommend that you check out JupyterLab (https://github.com/jupyterlab/jupyterlab)—Jupyter's next generation Notebook interface.

Here, we're looking for specific bugs in the Jupyter Notebook codebase. If you think you've identified such a bug, you can continue opening your issue here. We'd appreciate if you include as much detail as possible, i.e. links to the offending code, snapshots of the UI issue, code-blocks with your console logs, etc.

If you're having issues installing Jupyter Notebook, or you're having another issue and don't know how to proceed, try the following:

1. scan the "What to do when things go wrong" (https://jupyter-notebook.readthedocs.io/en/stable/troubleshooting.html#what-to-do-when-things-go-wrong) page in our documentation to see if your question has already been answered

2. post your question on the Jupyter Notebook discourse channel (https://discourse.jupyter.org/c/notebook/31). There are many more people in the Jupyter community that engage on that channel.
-->

**Describe the bug**
Apparently, `nest_asyncio` a dependency of jupyter notebook, released 1.5.2 this morning. So when installing Jupyter it installs this new version, which cause an error when running `jupyter notebook`.

**To Reproduce**
1. Install jupyter in a new env, or upgrade nest_asyncio to 1.5.2 in an env with Jupyter installed already.
2. Run `jupyter notebook`

**Expected behavior**
Jupyter notebook should be launched, but it hits an error (see below)

**Screenshots**
```
╰─ jupyter notebook                                                                                                                                                                                                        ─╯
Traceback (most recent call last):
  File "/Users/tailaiw/sandbox/jupyter-bug/venv/bin/jupyter-notebook", line 8, in <module>
    sys.exit(main())
  File "/Users/tailaiw/sandbox/jupyter-bug/venv/lib/python3.9/site-packages/jupyter_core/application.py", line 264, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/Users/tailaiw/sandbox/jupyter-bug/venv/lib/python3.9/site-packages/traitlets/config/application.py", line 845, in launch_instance
    app.initialize(argv)
  File "/Users/tailaiw/sandbox/jupyter-bug/venv/lib/python3.9/site-packages/traitlets/config/application.py", line 88, in inner
    return method(app, *args, **kwargs)
  File "/Users/tailaiw/sandbox/jupyter-bug/venv/lib/python3.9/site-packages/notebook/notebookapp.py", line 2141, in initialize
    self._init_asyncio_patch()
  File "/Users/tailaiw/sandbox/jupyter-bug/venv/lib/python3.9/site-packages/notebook/notebookapp.py", line 2117, in _init_asyncio_patch
    nest_asyncio.apply()
  File "/Users/tailaiw/sandbox/jupyter-bug/venv/lib/python3.9/site-packages/nest_asyncio.py", line 14, in apply
    raise ValueError('Can\'t patch loop of type %s' % type(loop))
ValueError: Can't patch loop of type <class 'NoneType'>
```

**Comment by spiemonte** [comment]
I have the same issue, it appears also when using python 3.7.

**Comment by tailaiw** [comment]
@spiemonte I manually fixed nest_asyncio at 1.5.1 and it works.

**Comment by YiHui-Liu** [comment]
> 



> @spiemonte I manually fixed nest_asyncio at 1.5.1 and it works.

Roll back the version of nest_asyncio to 1.5.1?

**Comment by vitkl** [comment]
> 1.5.1

Also works for me

**Comment by tailaiw** [comment]
Fixed after `nest_asyncio` upgraded to 1.5.4.
